site/walkthrough.md

```diff
@@ -0,0 +1,109 @@
+---
+title: weave Walkthrough
+---
+
+## launch DNS by default, IP allocation and naming, both with sensible defaults, rendezvous peering, auto-attach
+
+````bash
+root@a:~# weave launch
+root@a:~# weave run --name=pong -d -ti ubuntu
+````
+
+Weave automatically picked an IP range that isn't used locally (this
+range can also be specified explicitly, as an argument to `weave
```

Comment:
I was well aware of that issue when I wrote this. The automatic picking is a convenience that is meant to work in most situations but not always. When two weave networks join up they could check whether they picked the same ranges. Though quite what they should do when that check fails I do not know. Hmm. Is there any reasonable approach here that *doesn't* require the user to specify a range?